pkg/vfs/vfs.go

```diff
@@ -616,7 +617,8 @@ func normalizeDocPatch(data, patch *DocPatch, cdate time.Time) (*DocPatch, error
 	}
 
 	if patch.UpdatedAt.Before(cdate) {
-		return nil, ErrIllegalTime
```

Comment:
@nono Mainly two reasons:
1. When the file is received by Bob's Cozy we don't provide its "time of creation" (but we could).
1. We thought that it would be more interesting as a recipient to know when the file was received compared to when it was created.